src/group/communication/app.js

```diff
@@ -1,9 +1,11 @@
 import {
 	MOCK_GROUP_COMMUNICATION,
-	MOCK_GROUP_COMMUNICATION_COMMENTS,
 	MOCK_GROUP_COMMUNICATION_FOLLOWERS,
 	MOCK_GROUP_COMMUNICATION_SUGGESTED_FOLLOWERS
 } from './api';
+import {
+	MOCK_COMMENTS_LIST,
+} from './../api';
```

Comment:
mmm no :) it's correct. it's up one folder from itself... maybe we could delete `./` instead